en/advanced-topics/engine-customization.md

```diff
@@ -112,4 +96,4 @@ To prevent the package from becoming too large, Creator excludes the native engi
         gulp gen-simulator
         ```
 
-        Once executed, a simulator project and simulator executable will be generated under the `engine-native/simulator` path, and the native simulator will be ready to run.
+        Once executed, a simulator project and simulator executable will be generated under the `cocos-engin/native/simulator` path, and the native simulator will be ready to run.
```

Comment:
Once executed, a simulator project and a simulator executable will be generated under the `cocos-engin/native/simulator` path, and the native simulator will be ready to run.